dataland-frontend/tests/e2e/utils/FakeFixtureUtils.ts

```diff
@@ -0,0 +1,11 @@
+import { faker } from "@faker-js/faker";
+
+/**
+ * Returns either the input string or undefined with a 50/50 chance
```

Comment:
For what is this needed?

Reply:
It's not my code, but it seems that this is a generic function used to simulate that fields in an LkSG dataset can sometimes be empty.
e.g. ``country: randomStringOrUndefined(faker.address.country())`` either fills the ``country`` field of a production site with a country (string) value, or with ``undefined``

Reply:
We have several functions like this in the fake fixture generation. The point is that we also want to have fake fixture values that are not set for fields that are not required. If we always set all values we would potentially miss bugs that are related to nit providing not required fields.

Reply:
Ok, understood. Thany you